dotcom-rendering/src/lib/liveblogAdSlots.ts

```diff
@@ -170,32 +249,44 @@ export const calculateApproximateElementHeight = (
 	const elementType = element._type as KnownBlockElementType;
 	const heightData = elementHeightDataMap[elementType];
 
-	let estimatedHeight = heightData.heightExcludingText + ELEMENT_MARGIN;
+	const heightExcludingText = isMobile
+		? heightData.heightExcludingTextMobile
+		: heightData.heightExcludingText;
 
-	// If the element has text that contributes to the height of the element, estimate
-	// the height of the text and increment the height
-	if (heightData.textHeight) {
-		const { lineHeight, lineLength } = heightData.textHeight;
-		const characterCount = heightData.text(element).length;
+	const estimatedHeightWithoutText = heightExcludingText + heightData.margin;
 
-		estimatedHeight += lineHeight * Math.ceil(characterCount / lineLength);
+	if (!heightData.text) {
+		return estimatedHeightWithoutText;
 	}
 
-	return estimatedHeight;
+	// If the element has text that contributes to the height of the element, estimate
```

Comment:
All of these comments are really helpful, and make the code much easier to read ✨